client/src/App.js

```diff
@@ -22,7 +23,12 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+       
+        <div className="App">
+        <Route exact path="/" component={MovieList} />
+        <Route path="/movies/:id" component={Movie} />
```

Comment:
great job adding the exact to the '/' route and not the movies dynamic route since it didnt need it